PROJECTS.md

```diff
@@ -0,0 +1,15 @@
+## Projects
+
+Portfolio projects are worth a collective weight of 45%.
+
+All projects are due before the start of class on the date specified. Please see individual assignments for submission instructions.
```

Comment:
this should say "Please see individual projects" not "Please see individual assignments"